docs/src/pages/en/usage/migration.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Migration
```

Comment:
This isn't technically correct for the rest of the community. Remember this is open-source and if this does grab traction most people will not have migrated from `@brandingbrand/flagship`

Reply:
Maybe we can rename it to `Troubleshooting`?